packages/core/saved-objects/core-saved-objects-migration-server-internal/src/zdt/model/stages/update_mapping_model_version.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import * as Either from 'fp-ts/lib/Either';
+import { throwBadResponse } from '../../../model/helpers';
+import type { ModelStage } from '../types';
+
+export const updateMappingModelVersion: ModelStage<
+  'UPDATE_MAPPING_MODEL_VERSIONS',
+  'DONE' | 'FATAL'
+> = (state, res, context) => {
+  if (Either.isLeft(res)) {
+    throwBadResponse(state, res as never);
+  }
+
+  return {
+    ...state,
+    controlState: 'DONE',
+  };
```

Comment:
Note: I'm currently moving to `DONE` so that the algo completes, and so I can run integration test against the part of the logic already implemented.

When we'll be added the next part (doc migration), this will transition to the next steps of the migration.